Celeste.Mod.mm/Mod/Everest/Everest.Events.cs

```diff
@@ -180,6 +181,12 @@ internal static void Complete(_Level level)
                     => OnComplete?.Invoke(level);
             }
 
+            public static class Session {
+                public static event Action<patch_Session.Slider, float?> OnSliderChanged;
+                internal static void SliderChanged(patch_Session.Slider slider, float? previous)
+                    => OnSliderChanged?.Invoke(slider, previous);
```

Comment:
I'd like this to give access to the `Session` object as well, either as an additional parameter or through `Slider`

Reply:
leaning towards putting it in the event, but i guess i could do it through the slider instance. any preference?